contracts/tests/LineFactoryAlt.t.sol

```diff
@@ -0,0 +1,80 @@
+import "forge-std/Test.sol";
```

Comment:
So we're merging in your code first and then we're merging Mo's factory tests? I remember u guys decided on some order the other day.